### DISCUSSION TITLE
Welcome to open-gpu-kernel-modules Discussions!

### DISCUSSION THREAD

**Discussion by aritger** [NVIDIA/open-gpu-kernel-modules#156] [Announcements]
## Welcome!
  We’re using Discussions as a place for announcements, questions, and other general discussion.
  * Please note the [Code of Conduct](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CODE_OF_CONDUCT.md).
  * Please note the [Security Reporting Policy](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/SECURITY.md).
  * Ask questions about the code base or anything else, on topic, you're wondering about.

**Comment by JoaoRuss0** [comment]


**Reply by master7720** [reply]
no way

**Comment by ghost** [comment]
Does NVIDIA have plans to open-source other GPU components (e.g. userland drivers, firmware, CUDA- & tensor-related modules)? It's awesome that the kernel modules are now FOSS and I think this decision has been a great step forward, but I've also heard a lot of concerns about proprietary blobs simply being moved from the drivers into the firmware, and I think there's still plenty of room to grow. A lot of the most important areas where NVIDIA GPU compute is involved like FEM, fluid dynamics, AI/ML, etc. would greatly benefit from a more open approach to the full GPU stack, not to mention that massive industry communities would be able to contribute to improving NVIDIA products.

**Reply by aritger** [reply]
We don't currently have plans for releasing the source to any of our other driver components.

**Comment by MollyMaclachlan** [comment]
> - Please note the [Code of Conduct](https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/CODE_OF_CONDUCT.md).
> - Please note the [Security Reporting Policy](https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/SECURITY.md).

FYI these two links don't work - they point to discussions/[document name], which don't exist. The correct links are:
- https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CODE_OF_CONDUCT.md
- https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/SECURITY.md

**Reply by aritger** [reply]
Fixed; thanks.

**Comment by edisionnano** [comment]
While this driver only supports Turing, Ampere and beyond will nvidia release reclocking firmware and other firmware for Nouveau to use for Maxwell and Pascal?

**Reply by sTiKyt** [reply]
Very curious about this one

**Comment by kairui1108** [comment]


**Reply by Greek** [reply]
OK

**Reply by caniko** [reply]
@Greek it is a reference to [this](https://www.youtube.com/watch?v=i2lhwb_OckQ)

**Reply by caniko** [reply]
Guys, lets not use the stick. Nvidia is a carrot company.
